util/uvmdvgen.py

```diff
@@ -104,12 +113,21 @@ def main():
         help=
         """Tests are now run with dvsim.py tool that requires a hjson based sim cfg.
              Setting this option will also result in the Makefile to be auto-generated (which is
-             the older way of building and running sims going through deprecation).""")
+             the older way of building and running sims going through deprecation)."""
+    )
 
     args = parser.parse_args()
     if args.agent_outdir == "name": args.agent_outdir = args.name
     if args.env_outdir == "name": args.env_outdir = args.name
 
+    if not args.has_ral:
```

Comment:
actually on second thoughts, rather than throw an error, is this better:
```python
  args.ral = args.is_cip or args.has_interrupts
```

Reply:
I think this would be good as well, probably a bit more elegant than exiting out of the script.

Reply:
Great! Just add a comment why we are doing this.

Reply:
Done.